Userland/Libraries/LibGfx/Filters/FastBoxBlurFilter.cpp

```diff
@@ -43,15 +43,8 @@ static void do_single_pass(int width, int height, size_t radius_x, size_t radius
     int div_x = 2 * radius_x + 1;
     int div_y = 2 * radius_y + 1;
 
-    Vector<u8, 1024> intermediate_red;
-    Vector<u8, 1024> intermediate_green;
-    Vector<u8, 1024> intermediate_blue;
-    Vector<u8, 1024> intermediate_alpha;
-
-    intermediate_red.resize(width * height);
-    intermediate_green.resize(width * height);
-    intermediate_blue.resize(width * height);
-    intermediate_alpha.resize(width * height);
+    Vector<Color> intermediate;
```

Comment:
Should we keep the previous optimization where we stack allocate ~1000 by default?

Reply:
In my testing it doesn't seem to provide any difference, but that certainly could be the test. In other cases maybe it does make a difference. I will put it back.

Reply:
It won't make much of  a difference most of the time (or in the box-shadow test) since 1024 is only good for images <= `32x32px`